CONTRIBUTING.md

```diff
@@ -209,28 +209,22 @@ by running the test suite in a Docker environment that simulates Swift on Linux.
 
 1. Install [Docker Desktop for Mac](https://www.docker.com/products/docker-desktop).
 
-2. Get the newest Swift Docker image with:
+2. Build Swift-DocC (see [Building Swift-DocC](#building-swift-docc)).
 
-    ```bash
-    docker pull swift
```

Comment:
Pulling `swift` is not a prerequisite to building the Dockerfile so I've removed this step.